src/lazy.js

```diff
@@ -1,6 +1,15 @@
 import { h, options } from 'preact';
 import { useState, useRef } from 'preact/hooks';
 
+const oldDiff = options.__b;
+options.__b = (vnode) => {
```

Comment:
When compat is imported this would exist twice, not a bad thing necessarily just wanted to point it out